code/modules/actionspeed/modifiers/status_effects.dm

```diff
@@ -6,3 +6,6 @@
 
 /datum/actionspeed_modifier/nooartrium
 	multiplicative_slowdown = 0.5
+
+/datum/actionspeed_modifier/power_chord
+	multiplicative_slowdown = -0.75
```

Comment:
this should probably share the kroncaine blacklist

otherwise, you're gonna get people being able to cuff each other in less than a second, which can open up some wack combos